index.html

```diff
@@ -56,142 +62,153 @@ <h2>Glad to see you!</h2>
-                    </a>
-                </li>
-                <li>
-                    <a href="https://github.com/Kandy-Peter">
-                        <img src="./images/Vector_github.png" alt="my github profil">
-                    </a>
-                </li>
-                <li>
-                    <a href="#">
-                        <img src="./images/Vector_medium.png" alt="my medium profil">
-                    </a>
-                </li>
-                <li>
-                    <a href="https://twitter.com/peter_kandy">
-                        <img src="./images/Vector_twitter.png" alt="my twitter profil">
-                    </a>
-                </li>
-                <li>
-                    <a href="#">
-                        <img src="./images/Vector_ang.png" alt="my angels profil">
-                    </a>
-                </li>
-            </ul>
+    <section class="clm-3" id="about">
+        <div class="abt-clm">
+            <div class="clm-pres">
+                <h1>About</h1>
+                <h2>Myself</h2>
+                <p>I’m a software developer! I can help you build a product , 
+                       feature or website Look through some of my work and experience!
+                        If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            </div>
+    
+            <div class="links">
+                <p>let's connect</p>
+                <ul>
+                    <li>
+                        <a href="https://www.linkedin.com/in/kandi-peter-a49590212/">
+                            <img src="./images/Linkedin_icon.png" alt="my linkedin profil">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://github.com/Kandy-Peter">
+                            <img src="./images/Vector_github.png" alt="my github profil">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./images/Vector_medium.png" alt="my medium profil">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="https://twitter.com/peter_kandy">
+                            <img src="./images/Vector_twitter.png" alt="my twitter profil">
+                        </a>
+                    </li>
+                    <li>
+                        <a href="#">
+                            <img src="./images/Vector_ang.png" alt="my angels profil">
+                        </a>
+                    </li>
+                </ul>
+            </div>
+            <button id="resume-btn">Get my resume</button>
         </div>
-        <button id="resume-btn">Get my resume</button>
+        
         <div class="scroll-menu">
             <ul>
                 <li class="about-list">Languages <i class="fas fa-chevron-right"></i></li>
```

Comment:
Please, Add the list of technologies used here. always follow the Figma design been given.
Your design
![nn](https://user-images.githubusercontent.com/53356820/144290448-1e4304dc-3177-4d09-824c-a54730d46d20.png)
Figma design
![nn1](https://user-images.githubusercontent.com/53356820/144290486-d2d97450-45b2-4ef2-b983-10bd4c0e24a5.png)

Reply:
As said, we don't need to implement any functionality that requires JavaScript and my last code reviewer agreed that. I didn't think that I should add and we don't yet start with Javascript. Hope, you get me